### PR TITLE
Fixes to links

### PR DESCRIPTION
**Pull request by internettrans**

### CODE REVIEW

File: docs/ecosystem-react.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: single-spa-react
 sidebar_label: React
 ---
 
-[![Build Status](https://travis-ci.org/single-spa/single-spa-react.svg?branch=master)](https://travis-ci.org/single-spa/single-spa-react)
+[![Build Status](https://travis-ci.com/single-spa/single-spa-react.svg?branch=master)](https://travis-ci.com/single-spa/single-spa-react)
```
Comment (internettrans) [inline docs/ecosystem-react.md:7]: Using travis-ci.com is now suggested for open source projects. I enabled it for the single-spa github org

**Comment by frehner** [comment]
Does this need merge in the latest from master? You’re reverting lots of stuff you did earlier

**Comment by internettrans** [comment]
My first PR had problems in it - this one fixes it. We should be using travis-ci.com, and sofe is still in the canopy github.

File: website/blog/2019-02-20-single-spa-inspector.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ authorImageURL: https://pbs.twimg.com/profile_images/1037162092963348480/1XLEV4M
...
+For a long time, Canopy has had the benefit of using a tool called [sofe inspector](https://github.com/CanopyTax/sofe-inspector) (note: this is an out-of-date version of it) to list, override, and interact with single-spa applications. There has always been a desire to [figure out how to share this tool so others can benefit as well](https://github.com/single-spa/single-spa/issues/151).
 
-With that in mind, I'm proud to announce an initial release for **[single-spa Inspector](https://github.com/single-spa/single-spa-inspector)**! single-spa Inspector is a Firefox and Chrome extension, much like React/Vue devtools, that allows you see and interact with your single-spa applications and configuration.
+With that in mind, I'm proud to announce an initial release for **[single-spa Inspector](https://github.com/CanopyTax/single-spa-inspector)**! single-spa Inspector is a Firefox and Chrome extension, much like React/Vue devtools, that allows you see and interact with your single-spa applications and configuration.
```
Comment (frehner) [inline website/blog/2019-02-20-single-spa-inspector.md]: this one should still be single-spa I believe

**Review by frehner — APPROVED** [review]

**Pull request merged**